packages/types/src/runner-config.ts

```diff
@@ -2,16 +2,29 @@
 
 import { RunnerContainerConfiguration } from "./sth-configuration";
 
-export type RunnerConfig = {
+type CommonSequenceConfig = {
+    id: string;
```

Comment:
```suggestion
    type: string;
    id: string;
```

Reply:
fixed

Reply:
Hmm... Have you committed this? I can't see type.

Reply:
yes I did :) On github when you click on a file url in a comment it points to old changes